Anki_App/public/public/home.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <link rel='stylesheet' href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Anki</title>
+</head>
+<body>
+    <header>
+        <div>
+            <a class="home_btn" href="home.html">Anki</a>
+        </div>
+        <nav class="navbar">
+            <div><a href="collections.html">Collections</a></div>
```

Comment:
Why do you need a wrapping `div` here?

Reply:
you are right. removed it and added margin to <a>

Reply:
<a>tag